types/react-star-ratings/react-star-ratings-tests.tsx

```diff
@@ -0,0 +1,18 @@
+import StarRatings from 'react-star-ratings';
```

Comment:
Use the commonjs syntax `import StarRatings = require('react-star-ratings')`.

Consumers of the package who want to use `import StarRatings from 'react-star-ratings'` will need to add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.